src/WSRequestHandler_Sources.cpp

```diff
@@ -965,6 +965,157 @@ void WSRequestHandler::HandleSetBrowserSourceProperties(WSRequestHandler* req) {
+ * @param {String (optional)} `to-scene-name` Name of the scene to create the item in. Defaults to the current scene.
+ * @param {Object} `item` item to delete (required)
+ * @param {String} `item.name` name of the scene item (prefer `id`, including both is acceptable).
+ * @param {int} `item.id` id of the scene item.
+ *
+ * @api requests
+ * @name DuplicateSceneItem
+ * @category sources
+ * @since unreleased
+ */
+void WSRequestHandler::HandleDuplicateSceneItem(WSRequestHandler* req) {
+    if (!req->hasField("item")) {
+        req->SendErrorResponse("missing request parameters");
+        return;
+    }
+
+    const char* fromSceneName = obs_data_get_string(req->data, "from-scene-name");
+    OBSSourceAutoRelease fromScene = Utils::GetSceneFromNameOrCurrent(fromSceneName);
+    if (!fromScene) {
+        req->SendErrorResponse("requested fromScene doesn't exist");
+        return;
+    }
+
+    const char* toSceneName = obs_data_get_string(req->data, "to-scene-name");
+    OBSSourceAutoRelease toScene = Utils::GetSceneFromNameOrCurrent(toSceneName);
+    if (!toScene) {
+        req->SendErrorResponse("requested toScene doesn't exist");
+        return;
+    }
+
+    OBSDataAutoRelease item = obs_data_get_obj(req->data, "item");
+    OBSSceneItemAutoRelease *referenceItem;
+    if (obs_data_has_user_value(item, "id")) {
+        referenceItem = (OBSSceneItemAutoRelease *)Utils::GetSceneItemFromId(fromScene, obs_data_get_int(item, "id"));
+        if (obs_data_has_user_value(item, "name") &&
+            obs_source_get_name(obs_sceneitem_get_source((obs_sceneitem_t *)referenceItem)) !=
+            obs_data_get_string(item, "name")) {
+            req->SendErrorResponse("Invalid sceneItem id/name combination");
+            return;
+        }
+    }
+    else if (obs_data_has_user_value(item, "name")) {
+        referenceItem = (OBSSceneItemAutoRelease *)Utils::GetSceneItemFromName(fromScene, obs_data_get_string(item, "name"));
+    }
+
+    OBSSourceAutoRelease fromSource = (obs_source_t *)obs_sceneitem_get_source((obs_sceneitem_t*)referenceItem);
+    OBSSourceAutoRelease *newSource = (OBSSourceAutoRelease *)obs_source_duplicate(fromSource, obs_source_get_name(fromSource), false);
+
+    OBSSceneItemAutoRelease newItem = obs_scene_add(obs_scene_from_source(toScene), (obs_source_t *)newSource);
+    obs_sceneitem_set_visible(newItem, obs_sceneitem_visible((obs_sceneitem_t *)referenceItem));
+
+    if (!newItem) {
+        req->SendErrorResponse("Error duplicating scenee item");
+    }
+    // consider returning the new item and scene name
+    req->SendOKResponse();
+}
+
+/**
+ * Reset a source item.
+ *
+ * @param {String (optional)} `scene-name` Name of the scene the source belogns to. Defaults to the current scene.
+ * @param {String} `item` Name of the source item.
+ *
+ * @api requests
+ * @name ResetSceneItem
+ * @category sources
+ * @since 4.2.0
+ */
+void WSRequestHandler::HandleResetSceneItem(WSRequestHandler* req) {
```

Comment:
This appears to be failing the build as it duplicates another method name already defined on WSRequestHandler.
```
WSRequestHandler_Sources.cpp:(.text+0x5664): multiple definition of `WSRequestHandler::HandleResetSceneItem(WSRequestHandler*)'
```

https://github.com/Palakis/obs-websocket/blob/811e3f8cc80890ba4688d941af060455480fe535/src/WSRequestHandler_SceneItems.cpp#L333-L366

Double check me, but it looks like these are indeed duplicates. You should be able to remove this definition and let it call the other definition (unless this is a better home file for this method).